content/blog/2023-02-28-ethereum-classic-is-a-commodity-ethereum-is-a-security/index.md

```diff
@@ -0,0 +1,140 @@
+
+These pools may filter who may stake and/or who may be part of their validator set, or may be the node operators themselves who validate blocks and transactions. 
+
+These few entities will be static, meaning they will seldom rotate from the industry because of the moat in their businesses, will act in coordination with the Ethereum Foundation, and all together combined will clearly form a common enterprise directing the future of the system.
+
+This level of centralization and the lack of the proof of work cryptographic stamps when producing blocks completely eliminate the chain selection point, thus the attributes of free exit and entry, permissionlessness, censorship resistance, and immutability in Ethereum, making it a common enterprise with traditional centralized features and controls.
+
+ETC remains a commodity because it is and will remain Proof of Work (PoW) for the foreseeable future. 
+
+Therefore, there aren’t and will never be enterprises or groups who will control ETC, and mining and owning the coin will always be totally separate activities. 
+
+ETC has no foundation directing the roadmap of the system, and producing or verifying blocks are free, decentralized, and merit based activities.
+
+### 3. A reasonable expectation of profit: 
+
+As said before, although both ETH and ETC may be used purely as units of exchange, they may also be used as a store of value for price appreciation, therefore both qualify as having a reasonable expectation of profit.
+
+### 4. Derived from the efforts of others: 
+
+For the same reasons that Ethereum is now a common enterprise it is also a system where the expectation of the success of the system, and thus its future profits, are derived from the efforts of others. 
+
+These “others” will be perfectly identifiable, as they will be 4 or 5 large staking pools and the Ethereum Foundation, and all together will act as a single enterprise. 
+
+The PoS system is not a free merit system as PoW but a totally collaborative partnership between the pools and the validators, and the validators are essentially the same as the pools because they are their submissive contractors.
+
+In the system, validators divide their tasks on a per block basis where one produces a block, then sends it to the other validators who vote on the block, and then send it to the the rest of the network who must accept them without objection.
+
+Because ETC is PoW and decentralized, miners may pop up or leave whenever they want from anywhere in the world, compete to build blocks in their complete isolation, send them to the rest of the network for verification, and get paid based on their sole merit. No other filters or conditions exist. 
```

Comment:
Line 76: Rewrite for clarity
```suggestion
Because ETC uses proof of work and is, therefore, decentralized, miners may pop up or leave whenever they want from anywhere in the world, compete to build blocks in complete isolation, send blocks produced to the rest of the network for verification, and get paid based on their sole merit. No other filters or conditions exist. 
```